### ISSUE TITLE
SyntaxError: invalid syntax : line 63 in ut.py file

### ISSUE THREAD

**Issue by maxcado** [jin10086/bnb-cli#1]
Hello,
First thank you for posting your code. I wanted to let you know about a syntax error on line 63 in ut.py file.

I get the following:
f"bnbcli account {account} -- trust-node --node=data-seed-pre-2-s1.binance.org:80 -o json".split())

The syntax error is at the json".split()).

**Comment by jin10086** [comment]
Try it with python3

**Comment by maxcado** [comment]
I've tried using python3 and now I get a bunch of other errors like such:

File "run.py", line 15, in <module>
 NameWithAddress = getNameWithAddress()
File "bnb-cli/ut.py", line 51, in getNameWithAddress
 d= getkeylist()
File "bnb-cli/ut.py", line 35, in getkey list
d = runcleos("bnbcli keys list -0 json".split())

then something about line 30, 
line 403 in /usr/lib/python3.6/subprocess.py
and line 1344 

Basically, there were more errors using python3.
